projects/packages/forms/src/blocks/contact-form/components/jetpack-integrations-modal/active-integrations/style.scss

```diff
@@ -0,0 +1,19 @@
+.active-integrations {
```

Comment:
Since this will end up at global scope in WP with all kinds of other plugins, let's have something less generic. Basically just prefix with `jetpack_` :-)